contrib/jupyter/build.py

```diff
@@ -5,24 +5,39 @@
 """
 
 import os, sys
-import orbitutils.run_all_unittests
+import build
+import subprocess
 
-file_directory = os.path.dirname(os.path.abspath(__file__))
-orbitutils_path = os.path.join(file_directory, "orbitutils")
 
-# We need to specify the "root" of the src tree. With this choice, we can then
-# import the proto as "import ClientProtos.capture_data_pb2.py"
-proto_path = os.path.join(file_directory, "../../src/")
+def install_orbitutils(current_directory):
+    subprocess.check_call(
+        [sys.executable, "-m", "pip", "install", "-e", current_directory])
```

Comment:
Nit: current_directory -> installation_directory or target_directory?

Reply:
The directory passed actually needs to be the directory that contains the setup.py file and also contains the "orbitutils" folder. I renamed to "orbitutils_setup_path" and added a comment.